app/api/v1/endpoints/trading.py

```
@@ -559,40 +606,53 @@ async def get_recent_trades(
-                "status": "pending",
+                "amount": Decimal("0.001"),
+                "price": Decimal("0"),
+                "time": "No trades yet",
+                "status": "demo",
                 "pnl": Decimal("0"),
-            },
+            }
         ]
-        return RecentTradesResponse(recent_trades=mock_recent_trades)
+        return RecentTradesResponse(recent_trades=demo_trades)
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)